initial_vel: 1.0                   # Initial velocity of the MAV.
v_max: 2.0                         # Maximum velocity of the MAV.
a_max: 2.0                         # Maximum acceleration of the MAV.
visualize: true                    # Whether to publish information for visualization in RViz.
curvature: 2.0                     # Radius of the arc around the pylons.
num_arc: 5                         # Number of points to put into the trajectory in one arc around a pylon.
num_straight: 4                    # Number of points to put into the trajectory in one straight segment 
                                   # between the pylons.
rate: 10                           # Determines the loop rate.
command: true                      # Determines whether the mav has to follow the trajectory or not.
distance: 1.0                      # Distance by which to seperate additional markers.
dimension: 3                       # Determines the spatial dimension of trajectory.